readme.md

```diff
@@ -65,7 +80,7 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
-
+Models make requests to the backend usually for json data and then the views render this data. This way you can keep sensitive data on the backend. 
 ```
```

Comment:
:+1:

We can also use views to add functionality to our app (e.g., via event listeners).